### PR TITLE
story component

### PR DESCRIPTION
**Pull request by jtaningco**
no pic yet

### CODE REVIEW

File: src/components/Layout/Layout.jsx [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import React from "react"
 import Navbar from "../Navbar"
 import Footer from "../Footer"
+import Stories from "../Stories"
```
Comment (Hjkun77) [inline src/components/Layout/Layout.jsx:11]: don't add it in the Layout.jsx

File: src/components/Layout/Layout.jsx [hunk 2]
```diff
@@ -20,6 +21,7 @@ const Layout = ({ children, seo, slug }) => {
       <Navbar slug={slug} />
       <SEO title={siteTitle} description={siteDescription} />
       {children}
+      <Stories />
```
Comment (Hjkun77) [inline src/components/Layout/Layout.jsx:24]: don't add it here

create and put it in

`src\templates\story-list-page\index.jsx`

File: src/components/Stories/Story.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                        <CardImage src="" />
+                        <CardBody>
+                            <CardInfo> {story.category} </CardInfo>
+                            <CardTitle> {story.title} </CardTitle>
+                            <CardInfo>By {story.author} | { story.date } </CardInfo>
+                        </CardBody>
+                    </CardContainer>
+                </Card>
+            )
+        })
```
Comment (Hjkun77) [inline src/components/Stories/Story.jsx:28]:
``` stories.map((story, index) => (
                <Card key={index}>
                    <CardContainer> 
                        <CardImage src="" />
                        <CardBody>
                            <CardInfo> {story.category} </CardInfo>
                            <CardTitle> {story.title} </CardTitle>
                            <CardInfo>By {story.author} | { story.date } </CardInfo>
                        </CardBody>
                    </CardContainer>
                </Card>
            ))``` instead

File: src/components/Layout/Layout.jsx (hunk 2 above)
Comment (Hjkun77) [reply src/components/Layout/Layout.jsx:24]: inside `src\templates\story-list-page\index.jsx`

add this
```
import React from 'react'
import Stories from '../../components/Stories';
import Layout from '../../components/Layout';

const StoryPage = () => {
const seo = {
siteTitle: 'Hello'.
siteDescription: 'World'
}
return (
<Layout path="/stories" seo={seo}><Stories /><Layout />
)
}

Comment (Hjkun77) [reply src/components/Layout/Layout.jsx:24]: refer to the other template pages for reference

**Review by Hjkun77 — CHANGES_REQUESTED** [review]
Please address these concerns hehehe ask me if u don't get it

File: src/components/Layout/Layout.jsx (hunk 1 above)
Comment (jtaningco) [reply src/components/Layout/Layout.jsx:11]: Got this boss haha

File: src/components/Stories/Story.jsx (hunk 3 above)
Comment (jtaningco) [reply src/components/Stories/Story.jsx:28]: ques lang regarding the difference ng two na yan boss

Comment (Hjkun77) [reply src/components/Stories/Story.jsx:28]: I removed the curly braces

**Pull request merged**